### PR TITLE
Bning0822 patch 1

### PR DESCRIPTION
**Pull request by bning0822**
Finished the "supported features" and "principle of operations" topics in the user guide

### CODE REVIEW

File: V1/Modbus/ModbusOverview.md [hunk 1]
```diff
@@ -12,6 +12,90 @@ The Modbus TCP EDS adapter communicates with any device conforming to the Modbus
...
+
+ The value with data conversion applied will always be converted to 32-bit float type to keep the precision of the conversion factor and conversion offset.
+
+## Principles of operation
+The following topics give you an operational overview of the Modbus EDS adapter, focusing on streams creation and error handling. 
+
+### Operational overview
+
+#### Adapter configuration
+In order for the Modbus EDS adapter to be ready for data collection, you need to configure the adapter in order to provide necessary information. For more details, see **Configuration of Modbus data source** and **Configuration of Modbus data selection** sections. For a proper configuration of the connector, configure the following:
```
Comment (osimarkrogers) [inline V1/Modbus/ModbusOverview.md]: Change to "In order to provide the necessary information for the EDS Modbus adapter to be ready for data collection, you need to configure the adapter."

Comment (osimarkrogers) [reply V1/Modbus/ModbusOverview.md]: Change to "To configure the adapter, complete the following:"

File: V1/Modbus/ModbusOverview.md [hunk 2]
```diff
@@ -38,14 +122,15 @@ The following parameters are available for configuring a Modbus data source.
 
 | Parameter                |Required       | Type      | Description  |
 |--------------------------|-----------|-----------|---------------------------------------------------|
-| **IpAddress**           | Required  | string    | The IP Address of the device from which the data is to be collected using the Modbus protocol. Host name is not supported. |
```
Comment (TinaUll) [inline V1/Modbus/ModbusOverview.md]: IP Address > IP **a**ddress

File: V1/Modbus/ModbusOverview.md [hunk 3]
```diff
@@ -38,14 +122,15 @@ The following parameters are available for configuring a Modbus data source.
...
-| **Port**                | Optional  | number | The TCP port of the target device that listens for and responds to Modbus requests. The value ranges from 0 to 65535. If not configured, the default TCP port is 502 (which is the default port for Modbus protocol). |
-| **StreamIdPrefix**      | Optional          | number | Parameter applied to all data items collected from the data source. If not configured, the default value will be the ID of the Modbus EDS adapter. The custom StreamIdPrefix has the highest priority.|
```
Comment (TinaUll) [inline V1/Modbus/ModbusOverview.md]: will be > is

File: V1/Modbus/ModbusOverview.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bning0822) [reply V1/Modbus/ModbusOverview.md]: The section is actual inside the same page, I am not sure if we need to add a link in this case

**Pull request merged**